obspy/core/stream.py

```diff
@@ -1563,6 +1565,18 @@ def slice(self, starttime=None, endtime=None, keep_empty_traces=False):
         :type keep_empty_traces: bool, optional
         :param keep_empty_traces: Empty traces will be kept if set to ``True``.
             Defaults to ``False``.
+        :type nearest_sample: bool, optional
+        :param nearest_sample: If set to ``True``, the closest sample is
+            selected, if set to ``False``, the next sample containing the time
```

Comment:
The word "next" is a bit confusing, because it indicates "after", but for the start time (as in the example below), it seems to mean "before".